latch_cli/snakemake/workflow.py

```diff
@@ -1051,19 +957,20 @@ def _serialize_pod_spec(self, settings: SerializationSettings) -> Dict[str, Any]
             containers.append(V1Container(name=self.task_config.primary_container_name))
 
         final_containers = []
+        sdk_default_container = super().get_container(settings)
         for container in containers:
             # In the case of the primary container, we overwrite specific container attributes with the default values
             # used in the regular Python task.
             if container.name == self.task_config.primary_container_name:
-                sdk_default_container = super().get_container(settings)
-
                 container.image = sdk_default_container.image
                 # Spawn entrypoint as child process so it can receive signals
                 container.command = [
                     "/bin/bash",
                     "-c",
                     (
-                        "exec 3>&1 4>&2 && ("
+                        "exec 3>&1 4>&2 && latch cp"
+                        f" latch:///.snakemake_latch/workflows/{self.wf.name}/entrypoint.py"
+                        " /opt/latch/latch_entrypoint.py && ("
```

Comment:
this makes sm executions not reproducible bc if you overwrite the entrypoint, then relaunch the execution, the result might be different. theres also a race condition between multiple concurrent jit executions too since they all write to the same entrypoint

need to namespace this by version/execution name as well